### PR TITLE
Create a "secret" organization view intended for super-admins

### LINKED ISSUE #1137: 'Secret' Organization view with possibility to delete projects
Super users (admins of SAVE) should see the list of all organizations and have possibility to delete some dummy organizations.

### PR DESCRIPTION
**Pull request by 0x6675636b796f75676974687562**
Closes #1137.

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    )
+
+    init {
+        state.organizations = mutableListOf()
+    }
+
+    override fun componentDidMount() {
+        super.componentDidMount()
+
+        scope.launch {
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt:120]: you can make a request and set state directly in the FC

Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt:120]: check `useRequest {}`

**Review by akuleshov7 — APPROVED** [review]

**Review by sanyavertolet — APPROVED** [review]

**Review by Cheshiriks — APPROVED** [review]

**Review by nulls — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+        private const val INITIAL_PAGE_SIZE = 10
+    }
+}
+
+/**
+ * The state of the [OrganizationAdminView] component.
+ *
+ * @see OrganizationAdminView
+ */
+internal external interface OrganizationAdminState : State {
```
[the remark this answers is not in the document]
Comment (0x6675636b796f75676974687562) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt:228]: Will refactor and implement in a separate PR once there's a clear understanding how to use `tableComponent() `together with `FC<*>`.

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt (hunk 1 above)
Comment (0x6675636b796f75676974687562) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/OrganizationAdminView.kt:120]: Will refactor and implement in a separate PR once there's a clear understanding how to use `tableComponent() `together with `FC<*>`.

**Pull request merged**